test/shrinkwrap-specs.js

```diff
@@ -0,0 +1,23 @@
+// these are extra unit tests to ensure that appium is set up correctly for publishing
+
+import _ from 'lodash';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.use(chaiAsPromised);
+var should = chai.should();
+
+describe('shrinkwrap checks', () => {
+  it('shrinkwrap file should exist', async () => {
+    require('../../npm-shrinkwrap.json');
+  });
+
+  it('shrinkwrap should not include fsevents', async () => {
+    // fsevents is an optional dep that only works on Mac.
+    // if it's in shrinkwrap, non-Mac hosts won't be able to install appium
+    let shrinkwrap = require('../../npm-shrinkwrap.json');
+    should.exist(shrinkwrap.dependencies);
+    _.values(shrinkwrap.dependencies).length.should.be.above(10);
+    should.not.exist(shrinkwrap.dependencies.fsevents);
```

Comment:
is there a way to pull `fsevents` out of `package.json`'s optional dependencies? Rather than hard coding it?